docs/troubleshooting/faqs/TypeScript.mdx

```diff
@@ -52,3 +52,27 @@ You can use ESLint to enforce good uses of both ESLint and TypeScript comment di
 - The [`@eslint-community/eslint-plugin-eslint-comments`](https://eslint-community.github.io/eslint-plugin-eslint-comments) plugin can enforce general ESLint comment best practices, including requiring descriptions
 
 :::
+
+## How should I handle reports that conflict with [`verbatimModuleSyntax`](https://www.typescriptlang.org/tsconfig/#verbatimModuleSyntax)?
+
+Several TypeScript options impact how imports and exports are handled in your project, including
```

Comment:
[Style] Nitty nit, just noticed that most of our lists prefixed with a sentence have that sentence end with a `:`

```suggestion
Several TypeScript options impact how imports and exports are handled in your project, including:
```